src/no/runsafe/nchat/channel/ChannelManager.java

```diff
@@ -49,11 +49,14 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
 		if (cmp == 0)
 			return null;
 
-		if (ignoreHandler.playerIsIgnoring(player1Name, player2Name))
-			return null;
+		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
+		{
+			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2))
```

Comment:
could playerIsIgnoring be made to check both ways? if not, could we merge these two ifs?